src/js/components/Diagram/index.d.ts

```diff
@@ -6,8 +6,32 @@ export type DiagramConnectionAnchor = 'center' | 'vertical' | 'horizontal';
 export type DiagramConnectionType = 'direct' | 'curved' | 'rectilinear';
 
 export interface DiagramProps {
+  animation?:
+    | boolean
+    | 'pulse'
+    | 'draw'
+    | {
+        type?: 'pulse' | 'draw';
+        // looking into it, but delay seems to only work when delay <= 1000
+        delay?: number;
+        duration?: number;
+        // currently, the only animation Diagram has that utilizes size is for "pulse"
```

Comment:
This comment should be placed on doc.js "size is only applicable when using `pulse`"

Reply:
Recent commit now has this comment in `doc.js` for `size` attribute's description.